src/index.css

```diff
@@ -80,3 +80,44 @@ code {
 .custom-date-filter-input:hover {
   cursor: pointer;
 }
+
+.st0 {
+  fill: none;
+  stroke: #000000;
+  stroke-miterlimit: 10;
+}
+
+.st1 {
+  font-family: 'MyriadPro-Regular';
+}
+
+.st2 {
+  font-size: 85px;
+}
+
+.svgPlot {
+  enable-background: new 0 0 910 577;
+  margin: auto;
+}
+
+.svgPlot3 {
+  enable-background: new 0 0 910 577;
+  margin: auto;
+  padding: 100px;
+}
+
+.svgPlotNumber {
+  white-space: pre;
+  cursor: pointer;
+}
+.svgPlotNumber:hover {
+  fill: royalblue;
+}
+
+.svgPlotText {
+  white-space: pre;
+  fill: darkred;
+  font-family: Arial, sans-serif;
+  font-weight: bolder;
+  font-size: 35px;
+}
```

Comment:
Would it also be possible to define the styles somewhere else and not in the global index.css? If that's not easily possible, it would be great to add a bit of documentation to the file (so far, we haven't done a good job at documenting the styles - sorry for that!).

Reply:
Would a css file inside the components directory be ok?

Reply:
I think this would be an improvement!